### ISSUE TITLE
Release 1.9.1

### ISSUE THREAD

**Issue by BGmot** [ansible-collections/community.zabbix#869]
We need to release 1.9.1

**Comment by BGmot** [comment]
@D3DeFi please review the PR, I've set the date for Jan 6th (tomorrow in my time zone).

**Comment by BGmot** [comment]
@D3DeFi what should I do now after https://github.com/ansible-collections/community.zabbix/pull/873 is merged?
Merge main into 1.9.1prep, re-run antsibull-changelog and create new PR?

**Comment by D3DeFi** [comment]
> @D3DeFi what should I do now after #873 is merged? Merge main into 1.9.1prep, re-run antsibull-changelog and create new PR?

Yep, but try to rebase 1.9.1prep on top of main and then re-run antsibull-changelog

**Comment by BGmot** [comment]
I tried re-basing
```
$ git rebase upstream/main
First, rewinding head to replay your work on top of it...
Applying: Add missing changelog fragments.
Applying: Fix lint error.
Applying: galaxy.yml updated with new version.
Using index info to reconstruct a base tree...
M       galaxy.yml
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: README.md updated with new version.
Using index info to reconstruct a base tree...
M       README.md
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: CHANGELOG updated.
Using index info to reconstruct a base tree...
M       CHANGELOG.rst
M       changelogs/.plugin-cache.yaml
M       changelogs/changelog.yaml
A       changelogs/fragments/845-rhel-9-support.yml
A       changelogs/fragments/850-proxy-serverport.yml
A       changelogs/fragments/fix-httpapi-http-auth.yml
Falling back to patching base and 3-way merge...
Removing changelogs/fragments/859-remove-warn-arg.yml
Removing changelogs/fragments/857-deprecation-warnings.yml
Auto-merging changelogs/changelog.yaml
CONFLICT (content): Merge conflict in changelogs/changelog.yaml
Auto-merging CHANGELOG.rst
CONFLICT (content): Merge conflict in CHANGELOG.rst
error: Failed to merge in the changes.
Patch failed at 0005 CHANGELOG updated.
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
```
I can fix conflicts of course but we are trying to apply commits that were "fixed" by other "later" commits and later if we go this way and merge 1.9.1prep into main we'll see the same commit messages twice. In my opinion re-base is not good here, merge is more appropriate. What do you think?

**Comment by D3DeFi** [comment]
hmm, then just new branch from the main one perhaps? If that is not correct then feel free to merge, but I would ditch the old branch

**Comment by BGmot** [comment]
Delete 1.9.1prep and create new one with the same name or new branch with let's say 1.9.1prep1 ?

**Comment by D3DeFi** [comment]
that is up to you, it does not matter :)

**Comment by BGmot** [comment]
Agree. End result is the same. Thanks.

**Comment by BGmot** [comment]
Antsibull-changelog updated only date which is ok I think. I set release date for tomorrow (just in case).
